operatorcourier/validate.py

```diff
@@ -232,43 +240,49 @@ def _pkgs_validation(self, bundleData):
 
         pkgs = bundleData[self.pkgsKey]
 
-        for pkg in pkgs:
-            if "packageName" in pkg:
-                logger.info("Evaluating package %s", pkg["packageName"])
-            else:
-                self._log_error("packageName not defined.")
-                valid = False
+        num_pkgs = len(pkgs)
+        if num_pkgs != 1:
+            logger.error('Only 1 package is expected to exist per bundle, but got %d.', num_pkgs)
```

Comment:
This should use the _log_error method

Reply:
Thanks @awgreene. Great catch! Updated the PR.